extra/dist/world.py

```diff
@@ -1,23 +1,24 @@
-from typing import Optional
+import ctypes
 from extra import dist
 from multiprocessing import shared_memory
 from tinygrad.helpers import DEBUG, colored, getenv
 from tinygrad.lazy import LazyBuffer
 from tinygrad.runtime.lib import RawBuffer, RawBufferCopyInOut
```

Comment:
This doesn't work anymore due to this.

Reply:
I just updated this file as well to match calls to hip. Should i revert since it's not supported any more?

Reply:
meh, it's fine